### ISSUE TITLE
Accessibility review of Figma annotations for Templates, Patterns and Forms library

### ISSUE THREAD

**Issue by babsdenney** [department-of-veterans-affairs/vets-design-system-documentation#2803]
## Description
@LWWright7 completed adding accessibility annotations for all forms and templets and needs @rsmithadhoc to add any other a11y annotations as needed 

## Details

- This is a continuation of this [ticket #2545 ](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/2545)
-  Page to review annotations - [Figma Review template](https://www.figma.com/file/4A3O3mVx4xDAKfHE7fPF1U/VADS-Templates%2C-Patterns%2C-and-Forms?type=design&node-id=2988-66785&mode=design)

## Tasks
- [ ] Review [Figma Review templates](https://www.figma.com/file/4A3O3mVx4xDAKfHE7fPF1U/VADS-Templates%2C-Patterns%2C-and-Forms?type=design&node-id=2988-66785&mode=design) accessibility annotations
- [ ] Request any changes made from DS Designers

## Acceptance Criteria
- [ ] @rsmithadhoc has approved annotations

**Comment by caw310** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2803?planning-poker) with Zenhub @rsmithadhoc

**Comment by LWWright7** [comment]
@caw310 this is not a design ticket, I believe it should be a11y ticket for @rsmithadhoc

**Comment by rsmithadhoc** [comment]
@babsdenney @danbrady @LWWright7 I believe this is all set. 

I left comments in Figma for individual element updates, I only have read/comment access, so the annotations themselves will need to be added. I went through all the pages and commented on ones that looked finished or at least in progress, but let me know if I missed anything important or if there are questions, but I'll probably have to revisit at some point anyway as things get added and edited. Feel free to resolve any comments.

I only left one comment on the desktop version, but the same annotation is needed for the mobile view, just didn't want to flood it with too many comments.

For components that have a customizable heading level, such as "Process list" or "Accordion", I commented with what that should be. For others that don't have customizable heading levels, such as "Need help?" or "Statement of truth", I did not leave any comment on. 

Thanks!
